src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (a, b) {
+      return a + b;
+    },
+
+    subtract: function (a, b) {
+      return a - b;
```

Comment:
[CODE KNOWLEDGE] - Use the shorthand method syntax for the `subtract` method as well. Replace `subtract: function (a, b) { return a - b; }` with `subtract(a, b) { return a - b; }`.